docs/content/preview/api/ysql/the-sql-language/statements/streaming_create_repl_slot.md

```diff
@@ -0,0 +1,58 @@
+---
+title: CREATE_REPLICATION_SLOT statement [YSQL]
+headerTitle: CREATE_REPLICATION_SLOT
+linkTitle: CREATE_REPLICATION_SLOT
+description: Use the CREATE_REPLICATION_SLOT statement to create a replication slot.
+menu:
+  preview:
+    identifier: streaming_create_repl_slot
+    parent: statements
+type: docs
+---
+
+## Synopsis
+
+Use the `CREATE_REPLICATION_SLOT` statement to create a replication slot.
+
+## Syntax
+
+{{%ebnf%}}
+  create_replication_slot
+{{%/ebnf%}}
+
+## Semantics
+
+### *slot_name*
+
+The name of the replication slot. The name of the replication slot must be **unique across all databases**.
+
+### *output_plugin*
+
+The name of the output plugin used for logical decoding.
+
+### NOEXPORT_SNAPSHOT / USE_SNAPSHOT
```

Comment:
We had discussed about not documenting the USE_SNAPSHOT option. We just document the NOEXPORT_SNAPSHOT